bin/build-status

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use 5.010;
+
+use FindBin '$Bin';
+use lib "$Bin/../lib";
+use CPAN2OBS qw/ debug info prompt /;
+use XML::Simple qw/ XMLin /;
+use YAML::XS;
+
+use Getopt::Long;
+GetOptions(
+    "project=s" => \my $project,
+    "project-prefix=s" => \my $project_prefix,
+    "yaml" => \my $as_yaml,
+    "lastbuild" => \my $lastbuild,
+    "repo=s" => \my $repo,
+    "arch=s" => \my $arch,
+);
+$arch ||= 'x86_64';
+$repo ||= 'openSUSE_Tumbleweed';
+my $apiurl = "https://api.opensuse.org";
+#$project ||= "home:tinita:cpan-mirror:CPAN-";
+#$project_prefix ||= "home:tinita:cpan-mirror:CPAN-";
+#$project ||= "devel:languages:perl:CPAN-";
+
+my $cpan2obs = CPAN2OBS->new({
+    apiurl => $apiurl,
+});
+
+
+my %counts;
+
+if ($project_prefix) {
+    my @letters = ('A' .. 'Z');
+
+    if (@ARGV) {
+        @letters = map uc, @ARGV;
```

Comment:
I'd prefer to see only see the block form of `map`.

Reply:
hmm, I prefer this for simple statements

Reply:
This is one of the things in Perl Best Practices where i completely agree with Damian. The expression form doesn't scale well and is pretty much always harder to read.